core/src/main/java/org/apache/jsieve/tests/Header.java

```diff
@@ -199,10 +202,10 @@ protected boolean match(String comparator, String matchType,
             }
         // Iterate over the header values looking for a match
         boolean isMatched = false;
-        Iterator headerValuesIter = headerValues.iterator();
+        Iterator<String> headerValuesIter = headerValues.iterator();
         while (!isMatched && headerValuesIter.hasNext()) {
```

Comment:
BTW you are commenting outdated commit of an other pull request. Sorry. https://github.com/linagora/james-jsieve/pull/9